modules/apps/fragment/fragment-renderer-react-impl/src/main/java/com/liferay/fragment/renderer/react/internal/model/listener/FragmentEntryLinkModelListener.java

```diff
@@ -191,9 +206,10 @@ private void _updateNPMRegistry(
 	private static final Log _log = LogFactoryUtil.getLog(
 		FragmentEntryLinkModelListener.class);
 
+	private static BundleContext _bundleContext;
```

Comment:
@rafaprax Please just use a Snapshot to simplify the related logic.

Reply:
cc @regiCesar21